policy/Azure-Proactive-Resiliency-Library-v2/azurerm/web/azurerm_web_serverfarms_migrate_app_service_to_availability_zone_support.rego

```diff
@@ -0,0 +1,14 @@
+package Azure_Proactive_Resiliency_Library_v2
+
+import rego.v1
+
+valid_azurerm_web_serverfarms_migrate_app_service_to_availability_zone_support(resource) if {
+    resource.values.zone_balancing_enabled == true
+}
+
+deny_migrate_service_plan_to_availability_zone_support contains reason if {
+    resource := data.utils.resource(input, "azurerm_service_plan")[_]
+    not valid_azurerm_web_serverfarms_migrate_app_service_to_availability_zone_support(resource)
+
+    reason := sprintf("Azure-Proactive-Resiliency-Library-v2/migrate_service_plan_to_availability_zone_support: '%s' `azurerm_service_plan` must have a `zone_balancing_enabled` attribute set to true: https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/azure-resources/Web/serverFarms/#migrate-app-service-to-availability-zone-support", [resource.address])
```

Comment:
Need to address either `zone_balancing_enabled` is true, nor `sku_name` is not startswith `I` nor `P`.